### DISCUSSION TITLE
Background color.

### DISCUSSION THREAD

**Discussion by bert269** [kalkih/mini-graph-card#976] [Q&A] [answered]
I cannot seem to find a way to change the background color of a card. Is this possible?
color: does not seem to work on the same level as the card itself. It works for entities.
But how do I change the background color of the whole card?

Thanks

**Comment by ildar170975** [comment, marked as answer]
Changing background is only possible by card-mod.
For a particular solution: goto HA community -> card-mod thread -> 1st post -> link at the bottom -> styles for mini-graph-card

Please close this Discussion if you find the solution.

**Comment by bert269** [comment]
Thanks - that did the trick!. Appreciated!
